resources/views/admin/edit_category.blade.php

```diff
@@ -0,0 +1,37 @@
+@extends('layouts.app')
+
+@section('title', 'Edit Category')
+
+@section('content')
+<div class="container">
+    <div class="row">
+        <div class="col-md-3">
+            @include('adminsidebar')
+        </div>
+        <div class="col-md-9">
+            <div class="card">
+                <div class="card-header">Edit Category</div>
+
+                <div class="card-body">
+                    <form method="POST" action="{{ route('admin.updateCategory', ['category' => $category]) }}">
+                        @csrf
+                        @method('PUT')
+
+                        <div class="form-group">
+                            <label for="name">Category Name:</label>
+                            <input type="text" class="form-control" id="name" name="name" value="{{ $category->name }}" required>
```

Comment:
use required from request validation and make sure add old('name') to retain old value